TestCases/disc_adj_rans/naca0012/naca0012.cfg

```diff
@@ -0,0 +1,247 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: NACA0012 Laminar simulation (Re 5000)                      %
+% Author: Carlos Ilario                                                        %
+% Institution: Stanford University                                             %
+% Date: Mar 16, 2017                                                           %
+% File Version 5.0.0 "Raven"                                                   %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Please revise the description which seems incorrect, and try to highlight the features that this testcase exercises, both in its name and in the config file (by providing extra description or grouping the important stuff at the top). I also recommend trying to keep the config as clean as possible, if some options are not needed (e.g. RK coefficients) remove them as that can generate confusion for new users.

Reply:
Okay yes, I corrected the misleading description of the test case and removed unnecessary options.